src/odb/src/db/dbBlock.cpp

```diff
@@ -978,11 +974,18 @@ dbOStream& operator<<(dbOStream& stream, const _dbBlock& block)
   stream << block._inst_hash;
   stream << block._module_hash;
   stream << block._modinst_hash;
+
   if (db->isSchema(db_schema_update_hierarchy)) {
-    stream << block._modbterm_hash;
-    stream << block._moditerm_hash;
-    stream << block._modnet_hash;
-    stream << block._busport_hash;
+    if (!db->isSchema(db_schema_db_remove_hash)) {
+      dbHashTable<_dbModBTerm> unused_modbterm_hash;
+      dbHashTable<_dbModITerm> unused_moditerm_hash;
+      dbHashTable<_dbModNet> unused_modnet_hash;
+      dbHashTable<_dbBusPort> unused_busport_hash;
+      stream << unused_modbterm_hash;
+      stream << unused_moditerm_hash;
+      stream << unused_modnet_hash;
+      stream << unused_busport_hash;
+    }
```

Comment:
We do backwardly compatible reads but not writes.

Reply:
ok, removed.